sentinel/sentinel.go

```diff
@@ -102,6 +102,7 @@ type Client struct {
 	poolSize    int
 	masterPools map[string]*pool.Pool
 	subClient   *pubsub.SubClient
+	dialFunc    pool.DialFunc
```

Comment:
A comment here explaining why `pool.DialFunc` is being stored instead of this package's `DialFunc` would be helpful, even with your comment in the thread I was confused when I first looked at this =/